scripts/Jenkinsfile_batch_install_test

```diff
@@ -59,6 +59,10 @@ node {
                 actual_testdir_name = "validatecluster"
               }
 
+              if (test_name == "validatecluster_arm"){
```

Comment:
I think we can add OR statement for this test and above as they are referencing the same test dir
```
if (test_name == "docker_cri") || (test_name == "validatecluster_arm") {
  actual_testdir_name = "validatecluster"
}
```

Reply:
fixed